src/components/signup/SignUp.js

```diff
@@ -0,0 +1,87 @@
+import "./SignUp.css";
+import { FaTwitter } from "react-icons/fa";
+
+const SignUp = () => {
+  return (
+    <div className="signup-conatiner">
+      <div className="signup-split signup-left">
+        <div className="signup-centered">
+          <h2>Twitter</h2>
+          <FaTwitter className="singup-logo" />
+        </div>
+      </div>
+
+      <div className="signup-split signup-right">
+        <h3 className="signup-form-header">Signup</h3>
+        <form className="signup-centered">
+          <div className="signup-input-container">
+            <label htmlFor="name" className="signup-label">
+              <input
+                type="text"
+                className="signup-input"
+                placeholder="name"
+                required
+              />
+            </label>
+          </div>
+          <div className="signup-input-container">
+            <label htmlFor="username" className="signup-label">
+              <input
+                type="text"
+                className="signup-input"
+                placeholder="username"
+                required
+              />
+            </label>
+          </div>
+          <div className="signup-input-container">
+            <label htmlFor="email" className="signup-label">
+              <input
```

Comment:
This should have an `id` prop

Reply:
ok prop of the user. this will be passed for the functionality